charts/telemeter-reporting/templates/report-queries/nodes-by-type.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "nodes-by-type-raw"
```

Comment:
Nit: `nodes-by-instancetype-raw`? I'm thinking there may be other "types" besides instance types (might get confused with role?)